.github/workflows/nodejs.yml

```diff
@@ -166,6 +166,41 @@ jobs:
         if: ${{ steps.ts-cache.outputs.cache-hit != 'true' }}
         run: yarn build:ts
 
+  chromatic:
```

Comment:
this could be taken out of this PR, right?

Reply:
> Tempted to suggest that we merge this directly to main, as its pre-work for Chromatic but not directly related to the integration.

Hypothetically, yes this could go in straight to main. But it's already based off the Chromatic CI branch, and it'll be tough to untangle.

That said, it may be better to update to 7 sooner than later to avoid too many merge conflicts